portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -30,10 +30,12 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.lang.Integer;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
nit: update this comment now that this servlet is implemented

Reply:
will be updated in next git commit